db/migrate/20250207014022_add_currency_and_format_to_imports.rb

```diff
@@ -0,0 +1,6 @@
+class AddCurrencyAndFormatToImports < ActiveRecord::Migration[7.2]
+  def change
+    add_column :imports, :currency, :string
```

Comment:
I'm thinking we might want to rename this to `default_currency` (and remove the method that's already in the model) because a user is allowed to add a `currency` column in their CSV and assign a mapping to that.

If they explicitly provide us a currency column, we'll want to use that value first.  If not, we use this default value.

Reply:
Coming back to this after looking at #1814 and I'm realizing that we might need to entirely _remove_ this `currency` configuration column and just stick with the `number_format`.

Since the user can directly map one of their CSV columns to "Currency" and we're defaulting to the family currency anyways, I'm thinking this "Default currency" configuration option might cause more confusion than its worth. 

So in summary, I think we need to:

- Remove this `currency` column and the associated configuration field in the forms
- Put the `number_format` config field side by side with the existing currency mapping columns added in #1814

Reply:
For sure! Let me work on that!